### PR TITLE
Mobile skeleton

### PR DESCRIPTION
**Pull request by Jhaman1895**
Added a contact form to the mobile version of the Personal Portfolio project. This section was done in collaboration with my pair-programming partner Amol Charpe.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -169,5 +169,19 @@ <h4>LET'S CONNECT</h4>
...
+        If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it 
+      </p>
+      <form action="https://formspree.io/f/xbjwbgyj" method="POST">
+        <input type="text" id="name" name="name" maxlength="30" placeholder="name" required><br>
+        <input type="email" id='email' name="email" value="email" required><br>
+        <textarea  placeholder="Write your message here" name="message" id="message" cols="30" rows="10" maxlength="500" required></textarea>
+        <br>
+        <button type="submit">Get In Touch</button>
+      </form>
+    </div>
```
Comment (Ginohmk) [inline index.html:185]: - Please work on the background image for the contact form section, so it looks like that of the Figma design.
- Please add the horizontal small dash line at the footer, as it is on the Figma design 
- Please add a placeholder for your email field. 

##### Your design
Screenshot from 2022-06-08 07-42-48

##### Figma design 
Screenshot from 2022-06-08 07-44-41

**Review by Ginohmk — CHANGES_REQUESTED** [review]
Hi <STUDENT_MENTION>, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
you-are-near

## Highlights
- No lineter errors :+1: 
- Good commit message :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

- [Optional] PLease try to use proper padding and margin on all contents in your section, so it looks like the Figma design template. 
###### Reviewer 
- You can reach out to me on slack `@kanu mike chibundu` if you find any feedback confusing. 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Jhaman1895** [comment]
Improved the background image placement, added the horizontal line on the bottom, and added placeholder text for the email field in the contact form as mentioned in the previous review.

**Review by dasileker — APPROVED** [review]
### **Hi @Jhaman1895,** 

### **Congratulations! 🎉**

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 : 
 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**